### ISSUE TITLE
New Bookshelf Types...

### ISSUE THREAD

**Issue by datahead8888** [Secretchronicles/TSC#381]
Given we're seriously looking at a heavy library theme in the game, we probably need to make some more variations of bookshelves for library levels.  This will help with our visual variety in those levels.

**Comment by Quintus** [comment]
:+1:

**Comment by brianvanderburg2** [comment]
I think some of the book shelves could look old and worn out.  These could be good for dungeon based levels, where something new and shiny may not match a dungeon theme as good as a broken bookshelf/bookcase.

**Comment by datahead8888** [comment]
> I think some of the book shelves could look old and worn out. These could be good for dungeon based levels, where something new and shiny may not match a dungeon theme as good as a broken bookshelf/bookcase.

That's a really good idea.  I already put a bookshelf in my desert fortress level.  If we follow the idea of having ancient ruins in places (and maybe have references to an ancient civilization associated with the Great Library), worn out bookshelves makes sense in some places.

**Comment by Quintus** [comment]
Might be something for @DarkAceZ to look at as he was involved in the original bookshelf.

Vale,
Quintus

**Comment by DarkAceZ** [comment]
Were there plans for a customizable bookshelf? Where you could place empty bookshelves/books/other where you wanted?
We already have two different variations of the current shelf, one with Furball statues, one without. It would be rather excessive to have different colors/types of shelves, but have the same variation for each, and would be much more convenient to be able to have the statues as a separate placeable item.

**Comment by datahead8888** [comment]
> It would be rather excessive to have different colors/types of shelves, but have the same variation for each

By contrast, we have a lot more different kinds of pipes.  We were talking about having bookshelves that are much more different than just color.

> and would be much more convenient to be able to have the statues as a separate placeable item.

I don't follow - what do you mean?

**Comment by brianvanderburg2** [comment]
That could be good.  Have a modular bookshelf system similar to how the ground is, with top, middle, bottom, left, center, right pieces.  Then decorations, such as statues, could exist as separate images and be placed however they want on the bookshelf.

If a designer wants wants, they can make each shelf semi-passive, just the top shelf, none, or whatever else.

**Comment by datahead8888** [comment]
> That could be good. Have a modular bookshelf system similar to how the ground is, with top, middle, bottom, left, center, right pieces.

Yes, this sounds logical. It would also allow a shelf of arbitrary length to be assembled.  What would we do with the existing bookshelf - bust it up?

**Comment by Quintus** [comment]
Forum discussion: http://forum.secretchronicles.de/topics/441
